force_wfmanager/tests/test_wfmanager_task.py

```diff
@@ -84,6 +85,10 @@ def mock_show_error(*args, **kwargs):
     return args
 
 
+def mock_info_dialog(*args, **kwargs):
```

Comment:
i am growing excessively annoyed at the proliferation of these entities. We need to do something, but I am unsure about what.